source/Octopus.Cli/Commands/Runbooks/RunRunbookCommand.cs

```diff
@@ -116,7 +112,12 @@ public class RunRunbookCommand : ApiCommand, ISupportFormattedOutput
             options.Add<bool>("waitForRun", "[Optional] Whether to wait synchronously for deployment to finish.",
                 v => WaitForRun = true);
 
-            options.Add<bool>("progress", "[Optional] Show progress of the runbook run", v => { Progress = true; WaitForRun = true; NoRawLog = true; });
+            options.Add<bool>("progress", "[Optional] Show progress of the runbook run", v =>
```

Comment:
For future reference, I think we should avoid too many formatting changes like this in PRs - it just muddies the waters when reviewing.

Reply:
👍 Agreed.